junos/resource_interface_physical_test.go

```diff
@@ -96,6 +100,10 @@ func TestAccJunosInterfacePhysical_basic(t *testing.T) {
 							"ae_lacp", ""),
 						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
 							"ae_minimum_links", "0"),
+						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
+							"esi.identifier", "00:11:11:11:11:11:11:11:11:11"),
+						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
+							"esi.mode", "all-active"),
```

Comment:
Need to read len of `esi` list and read argument in first item.
```suggestion
						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
							"esi.#", "1"),
						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
							"esi.0.identifier", "00:01:11:11:11:11:11:11:11:11"),
						resource.TestCheckResourceAttr("junos_interface_physical.testacc_interfaceAE",
							"esi.0.mode", "all-active"),
```